.github/workflows/cpflow-deploy-review-app.yml

```
@@ -0,0 +1,382 @@
+            ].join("\n");
+
+            await github.rest.issues.updateComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              comment_id: commentId,
+              body
+            });
+
+      - name: Build Docker image
+        if: steps.config.outputs.ready == 'true' && steps.source.outputs.allowed == 'true' && (steps.check-app.outputs.exists == 'true' || steps.setup-review-app.outcome == 'success')
+        uses: ./.github/actions/cpflow-build-docker-image
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/cpflow-build-docker-image": yaml: line 62: could not find expected ':'

(action)